website/docs/guides/responses/text-search/relevance-score.js

```diff
@@ -1,17 +1,17 @@
 response = {
   cursor: {
-    id: Long('0'),
-    ns: 'new.books',
+    id: Long(0),
+    ns: 'cts.books',
```

Comment:
I don't really understand this part. Is there a reason why this is `cts.books`?

Reply:
It seems the default database is `cts`, see https://github.com/OpenDocDB/cts/blob/main/opendocdb-cts/main.go#L49. @noisersup could you ensure changing database name here doesn't impact where this file is referenced in the docs please :pray:

Reply:
Let's use "db", not "cts" or "new"